WooCommerce/Classes/Analytics/TracksProvider.swift

```diff
@@ -116,6 +116,7 @@ private extension TracksProvider {
         let exemptedEvents: Set<String> = [
             "application_opened",
             "application_closed",
+            "support_new_request_viewed",
```

Comment:
The support view opens in a modal within POS, so this event gets decorated as well with the `woocommerceios_pos_` bit unless we explicitly filter it out.

Reply:
> I think an allowed list would be safer in case we didn't catch any events that shouldn't be decorated, also left a comment in https://github.com/woocommerce/woocommerce-ios/pull/15006#issuecomment-2635488639

Any concern regarding performance when matching events against a "safe list"? As this list would be a way bigger string matching comparison than the "exempted list". I could be overthinking it entirely, so I'll capture how long it takes when making the changes 👍

Reply:
I've opened a new issue to make the switch from  exempted to allowed here: https://github.com/woocommerce/woocommerce-ios/issues/15058 I think we can make the switch once we get the final event list, so we can also confirm that all is there in one go